src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -15617,7 +15617,15 @@ namespace N1
    class A { public int Foo() { return 2; }}
 }
 ";
-            CreateCompilationWithMscorlib(text).VerifyDiagnostics();
```

Comment:
Also, the name of this test (which seems to be very hard to view in this PR) is ```CS7021ERR_NamespaceNotAllowedInScript```, which gave me the impression that the original intention was to test this diagnostic.